src/renderer/features/wallet-details/index.tsx

```diff
@@ -119,9 +122,16 @@ walletDetailsFeature.inject(walletActionSlot, ({ wallet }) => {
       title = t('walletDetails.common.hideButton');
     }
 
+    let onForget;
```

Comment:
This connection is forbidden because abstract wallet details feature shouldn't know about wallet connect. Add a comment about it here, please, we'll refactor it later